### PR TITLE
Devops 818 tfsec sarif action implementation for all directories

### PR DESCRIPTION
**Pull request by APB-s**
## What 🕵️ 🔍
- Tfscan will now scan only the repo that the PR refer to. (Modifying a .tf file in ionos directory will lead to a scan to this specific directory).

- Deletion of token

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/DEVOPS-818)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update(s) to the configuration of existing service(s)
- [ ] This change requires a documentation update
- [ ] Removed some deprecated application or functionality

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

## Checklist:

- [ ] I have made corresponding changes to the documentation.
- [ ] I have performed a self-review of my own code.
- [x] I did not commit anything sensitive like keys or secrets.
- [x] I added the corresponding labels.
- [x] I set the reviewers and the assignees.
- [ ] All open comments and discussions have been resolved.
- [x] My branch is following the name pattern: `JIRA-ID_Short_descripton`, example: `DEVOPS-000_some_meaningful_description`

### CODE REVIEW

**Review by APB-s — COMMENTED** [review]
I renamed the first file from tfsec_pr_commenter to tfsec_sarif_aws.yml wich is source of conflict.

I created 3 others file to scan the specific directory according the different PR.

**Review by APB-s — COMMENTED** [review]
- File name change.
- Created a specific file for each working directory containing .tf file
- Use of Tfsec  sarif security

File: .github/workflows/tfsec_sarif_ionos.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: tfsec-sarif-ionos-dev
+on:
+  pull_request:
+      - 'terraform/dev/*.tf'
```
Comment (abargiela) [inline .github/workflows/tfsec_sarif_ionos.yml:4]:
```suggestion
      - 'ionos/terraform/dev/*.tf'
```

File: .github/workflows/tfsec_sarif_ionos.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        'terraform/dev/*/*.tf'
```
Comment (abargiela) [inline .github/workflows/tfsec_sarif_ionos.yml:5]:
```suggestion
        'ionos/terraform/dev/*/*.tf'
```

File: .github/workflows/tfsec_sarif_ionos_prod.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: tfsec-sarif-ionos-prod
+on:
+  pull_request:
+      - 'terraform/prod/*/*/*.tf'
```
Comment (abargiela) [inline .github/workflows/tfsec_sarif_ionos_prod.yml:4]:
```suggestion
      - 'ionos/terraform/prod/*/*/*.tf'
```

**Review by abargiela — DISMISSED** [review]

**Review by abargiela — APPROVED** [review]

**Pull request merged**